app/src/main/java/com/android/unio/MainActivity.kt

```diff
@@ -48,14 +50,16 @@ fun UnioApp() {
   val db = FirebaseFirestore.getInstance()
   val associationViewModel: AssociationViewModel = viewModel(factory = AssociationViewModel.Factory)
 
+  val userRepositoryFirestore = UserRepositoryFirestore(Firebase.firestore)
+
   val context = LocalContext.current
 
   // Redirect user based on authentication state
   Firebase.auth.addAuthStateListener { auth ->
     val user = auth.currentUser
     if (user != null) {
       if (user.isEmailVerified) {
-        navController.navigate(Route.HOME)
+        navController.navigate(Screen.ACCOUNT_DETAILS)
```

Comment:
Later on, we will need to implement some logic that checks if the account details have already been given. This will ensure users that exit and relaunch the app during account details creation will still need to complete them before accessing the application.